src/components/PowerKnob.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { Silver } from "react-dial-knob";
+
+import { Table } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+
+export default function PowerKnob() {
+
+  // knob
+  const [value, setValue] = useState(0);
+  // probability functions ***MAP THESE LATER**
+  // Currently this isn't accurate as it calculates for each individual rarity
+  const common = Math.round((27.2/100) * ((value/200)))
+  const nine = Math.round((24.3/100)* ((value/200)))
+  const uncommon = Math.round((20.5/100)* ((value/200)))
+  const rare = Math.round((15/100)* ((value/200)))
+  const epic = Math.round((8.4/100)* ((value/200)))
+  const legendary = Math.round((3.8/100)* ((value/200)))
+  const mythical = Math.round((.8/100)* ((value/200)))
```

Comment:
Since everything uses (value/200) would be better to set that as a variable, and then use that variable instead of calculating it for each one. Also for your percentages you could just move the decimal place over two places instead of calculating and rounding. (27.2/100) = 0.272 and so on.